src/site/layouts/vet_center.drupal.liquid

```diff
@@ -131,10 +131,20 @@
                 typeOfCare = "other"
             %}
 
-            {% if fieldCcVetCenterFaqs.fetched.field_section_header.0.value != empty %}
-                <h2 id="how-different-than-clinic" class="vads-u-margin-top--0 vads-u-font-size--lg
-          small-screen:vads-u-font-size--xl vads-u-margin-bottom--2"> {{ fieldCcVetCenterFaqs.fetched.field_section_header.0.value }}</h2>
-                <p> {{ fieldCcVetCenterFaqs.fetched.field_section_intro.0.value }} </p>
+            {% if fieldCcVetCenterFaqs.fetched %}
+                {% if fieldCcVetCenterFaqs.fetched.field_section_header.0.value != empty %}
+                    <h2 id="how-different-than-clinic"
```

Comment:
I'm a bit concerned about hard-coding this `id`. We want this to change if the data changes. Can we derive it from the `field_section_header.0.value` instead, maybe?

Reply:
this id just to have an identifier when testing otherwise the id would be id="Some heading", now if we need to add aria labels we might need to pass field_section_header.0.value instead

Reply:
Make we should use a more generic label then?